README.md

```diff
@@ -103,6 +110,25 @@ PS C:\temp> .\httping.exe -url https://wormhole.network -count 5 -json
 {"host":"wormhole.network","httpVerb":"GET","hostHeader":"wormhole.network","seq":4,"httpStatus":200,"bytes":10991,"rtt":121.3327}
 {"host":"wormhole.network","httpVerb":"GET","hostHeader":"wormhole.network","seq":5,"httpStatus":200,"bytes":10991,"rtt":71.4523}
 ```
+
+#### Example 3
+
+Continuous monitoring of the connection quality,  
+```
+$ httping.exe -url http://detectportal.firefox.com/success.txt -count 0 -timeout 1000
+
+httping 0.9.1 - A tool to measure RTT on HTTP/S requests
```

Comment:
I need to consider bumping this to maybe 0.9.10 due to the change in behaviour (system proxy settings used by default)

Reply:
Hi. If you are using [Semantic versioning](https://semver.org/),  I'm thinking: `0.10.0` . Because changes don't just patch existing code, but make it a minor version update. Let me know what you think, thanks!

Reply:
On my phone at the moment and unsure how to add the change from here, would you mind bumping it to 0.10.0?

Reply:
Sure! Addressed in https://github.com/pjperez/httping/pull/15/commits/0cdef8e2afbbf0cbeb2e2f1311b78d23424ca7a8